docs/tests/integration/test_agentd/test_agentd_reconnection.md

```diff
@@ -1,3 +1,44 @@
+# Test AgentD reconnection
+
+## Overview
+
+These tests will check that, during enrollment, the agent re-establishes communication with the manager
```

Comment:
```suggestion
These tests will check if, during enrollment, the agent re-establishes communication with the manager
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca